Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Visual Extensions/VisualExtensionsCode.bind

```diff
@@ -0,0 +1,20 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+    mc:Ignorable="d">
+
+  <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Border Height="100"
+            Width="100"
+            Background="Purple"
+            extensions:VisualEx.CenterPoint="50,50,0"
```

Comment:
Thoughts about adding a "RelativeCenterPoint" extension that behaves similar to RenderTransform Centre Point? - as an Expression Animation so that it automatically updates the absolute value whenever associated visual changes size

Reply:
I think that's a great idea, makes a lot more sense then KeepCenterPointCentered. I just added a new property called NormalizedCenterPoint that takes values between 0 and 1, and removed the KeepCenterPointCentered.